third_party/ui/bower_components/angular-material/modules/closure/select/select.js

```diff
@@ -757,7 +757,7 @@ function SelectProvider($$interimElementProvider) {
       var focusedNode = centeredNode || optionNodes[0];
       if (focusedNode) {
         opts.focusedNode = focusedNode;
-        focusedNode.focus();
+        // focusedNode.focus();
```

Comment:
@bgrant0607 This is commented out to fix an issue in the angular directive where the first option remains in focus, even after you mouseover to a different option. It is fixed in the compiled source and is there for reference.